examples/src/bin/wifi_embassy_access_point.rs

```diff
@@ -54,6 +56,8 @@ macro_rules! mk_static {
     }};
 }
 
+const GW_IP_ADDR: &str = env!("GATEWAY_IP");
```

Comment:
Maybe we should make this `optional_env!` and unwrap to some default gateway of `192, 168, 2, 1` just to make the example a bit easier to run